velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1442,23 +1442,26 @@ struct CurrentDateFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
   const date::time_zone* timeZone_ = nullptr;
```

Comment:
timeZone_ member variable is no longer needed. Let's remove it.

Let's also make currentDate_ private.